src/electron/main.tsx

```diff
@@ -346,17 +350,25 @@ app.on('ready', () => {
 })
 
 const exitPyProc = () => {
-  pyProc.kill()
+  killPortProcess(31314)
+  killPortProcess(process.env.CLIMATE_TOKEN_DRIVER_PORT || 31314)
+  require('child_process').exec('taskkill /F /IM main.exe')
```

Comment:
Can this use the `${productionRunApp}` variable instead of hard-coding `main.exe`?

Reply:
Yes